src/main/resources/assets/js/components/Marathon.jsx

```diff
@@ -50,50 +51,46 @@ var Marathon = React.createClass({
     };
 
     Mousetrap.bind("esc", function () {
```

Comment:
This should get removed entirely. `esc` is not a UI standard for going back pages.